enviro/compute/fitting.py

```diff
@@ -357,18 +361,27 @@ def _get_fitting_values(sample, samples, name, dependency, index, number_of_inte
             order : (shape, loc, scale) (i.e. 0 -> shape)
 
         number_of_intervals : int,
-            number of distributions used to fit shape, loc, scale
+            Number of bins the data of this variable should be seperated for fits which depend
+            upon it. If the number of bins is given, the width of the bins is determined automatically.
 
-        Returns
-        -------
-        interval_centers :
+        bin_width : float,
```

Comment:
document whether, bin_width or number_of_intervals is used if both are supplied. Consider renaming number_of_intervals to bin_num, to emphasize the connection to bin_width.